tutorials/ui/gui_theme_type_variations.rst

```diff
@@ -0,0 +1,50 @@
+.. _doc_gui_theme_type_variations:
+
+Overview
+========
+
+When designing a user interface there may be times when a control node
+needs to have a different look than what is normally defined by the
+theme. Theme override properties of a node can be used. However if
+this change needs to be done multiple times the same work is being done
+multiple times, it also requires finding and editing every individual
+node with that look if it needs to be adjusted.
+
+As a solution to this problem there are theme type variations. These
+are used to create variations of control nodes in a way that is part
+of the theme.
+
+These can be used to redefine every aspect of a control node or only one.
+For example, if a button has a defined StyleBoxFlat for its normal state in
+the theme, and a button type variation only has a different font specified,
+any button using that type variation will still use the StyleBoxFlat defined
+for the theme.
+
+Creating a type variation
+-------------------------
+
+To create a type variation open the theme editor, then open the **Manage
+Theme Items** window. In the bottom left under **Add Type** enter a name
+for the  type variation then press the add button.
+
+Close the manage theme items window. In the theme editor on the right
+side click on the **Type** dropdown above the type editor. The type created
+in manage theme items should be listed, select it, then switch to the
+tab with a wrench and screwdriver icon.
```

Comment:
This is a very long way to add a new type. You can just click the plus icon next to the type selector, enter the name there, and press "Enter". It will create the type and remove the need to select it manually too.